x/lightclient/keeper/canonical_client.go

```diff
@@ -59,6 +59,10 @@ func (k Keeper) GetAllCanonicalClients(ctx sdk.Context) (clients []types.Canonic
 	return
 }
 
+func (k Keeper) expectedClient(ctx sdk.Context) ibctm.ClientState {
```

Comment:
why wrap this?

Reply:
because it's called in two places

Reply:
so why not just call `types.ExpectedCanonicalClientParams` in those two places?

Reply:
because you have to pass an argument